src/components/ShareModal.module.scss

```diff
@@ -1,13 +1,14 @@
 .container {
   max-width: 475px;
   width: 100%;
-  margin-bottom: 5px;
   display: flex;
   flex-direction: column;
   justify-content: center;
+  align-items: center;
   padding-bottom: 20px;
   padding-top: 20px;
   border-radius: 0px 0px 10px 10px;
+  box-shadow: none !important;
```

Comment:
What does this line accomplish? I commented it out and nothing visible changed (I saw a box-shadow get disabled in the dev tools)

Reply:
Thats because the box shadow gets hidden by the encapsulating div which is the `.ReactModal__Content`. If in the future someone changed the size of the div, all of a sudden he will find a box shadow for the card inside.